frontend/src/scenes/authentication/signup/verify-email/VerifyEmail.tsx

```diff
@@ -17,28 +19,75 @@ export const VerifyEmailHelpLinks = (): JSX.Element => {
     const { requestVerificationLink } = useActions(verifyEmailLogic)
     const { uuid } = useValues(verifyEmailLogic)
     const { openSupportForm } = useActions(supportLogic)
+    const [iNeedHelp, setINeedHelp] = useState(false)
+    const [checkListValues, setCheckListValues] = useState<boolean[]>([])
+
+    if (!iNeedHelp) {
+        return (
+            <LemonButton type="secondary" className="mt-2" onClick={() => setINeedHelp(true)}>
+                I need help
+            </LemonButton>
+        )
+    }
+
+    const checklist = ['Checked your spam folder', 'Checked with your IT department (if applicable)']
+
+    const handleChecklistChange = (index: number): void => {
+        const newCheckListValues = [...checkListValues]
+        newCheckListValues[index] = !newCheckListValues[index]
+        setCheckListValues(newCheckListValues)
+    }
+
+    const allChecked = checklist.every((_, index) => checkListValues[index])
 
     return (
-        <div className="flex flex-row gap-x-4">
-            <LemonButton
-                type="secondary"
-                className="mt-8"
+        <div className="w-full bg-bg-3000 p-4 rounded relative">
+            <IconX
+                className="absolute top-3 right-3 cursor-pointer"
                 onClick={() => {
-                    openSupportForm({ kind: 'bug', target_area: 'login' })
+                    setCheckListValues([])
+                    setINeedHelp(false)
                 }}
-            >
-                Contact support
-            </LemonButton>
-            {uuid && (
-                <LemonButton
-                    type="secondary"
-                    className="mt-8"
-                    onClick={() => {
-                        requestVerificationLink(uuid)
-                    }}
-                >
-                    Request a new link
-                </LemonButton>
+            />
+            <div className="flex flex-col justify-center">
+                <p className="text-left mb-2">Please confirm you've done the following:</p>
+                <div className="space-y-2">
+                    {checklist.map((item, index) => (
+                        <LemonCheckbox
+                            key={index}
+                            onChange={() => handleChecklistChange(index)}
+                            checked={checkListValues[index]}
+                            label={item}
+                            bordered
+                            size="small"
+                        />
+                    ))}
+                </div>
+            </div>
+            {allChecked && (
```

Comment:
I'd show these but disabled with a `disabledReason` saying something like "please confirm you've done the steps above". That way it's super clear what they need to do to activate the buttons.